config/crd/bases/operator.kyma-project.io_kymas.yaml

```diff
@@ -361,6 +361,355 @@ spec:
+                          type: string
+                        metadata:
+                          description: PartialMeta is a subset of ObjectMeta that
+                            contains relevant information to track an Object. see
+                            https://github.com/kubernetes/apimachinery/blob/v0.26.1/pkg/apis/meta/v1/types.go#L111
+                          properties:
+                            generation:
+                              description: A sequence number representing a specific
+                                generation of the desired state. Populated by the
+                                system. Read-only.
+                              format: int64
+                              type: integer
+                            name:
+                              description: 'Name must be unique within a namespace.
+                                Is required when creating resources, although some
+                                resources may allow a client to request the generation
+                                of an appropriate name automatically. Name is primarily
+                                intended for creation idempotence and configuration
+                                definition. Cannot be updated. More info: http://kubernetes.io/docs/user-guide/identifiers#names'
+                              type: string
+                            namespace:
+                              description: "Namespace defines the space within which
+                                each name must be unique. An empty namespace is equivalent
+                                to the \"default\" namespace, but \"default\" is the
+                                canonical representation. Not all objects are required
+                                to be scoped to a namespace - the value of this field
+                                for those objects will be empty. \n Must be a DNS_LABEL.
+                                Cannot be updated. More info: http://kubernetes.io/docs/user-guide/namespaces"
+                              type: string
+                          type: object
+                      type: object
+                    version:
+                      description: Channel tracks the active Version of the Module.
+                      type: string
+                  required:
+                  - fqdn
+                  - name
+                  - state
+                  - version
+                  type: object
+                type: array
+              state:
+                description: State signifies current state of Kyma. Value can be one
+                  of ("Ready", "Processing", "Error", "Deleting").
+                enum:
+                - Processing
+                - Deleting
+                - Ready
+                - Error
+                - ""
+                type: string
+            type: object
+        type: object
+    served: true
     storage: true
```

Comment:
Since v1beta1 is the new storage version which should by then be the default, what do you think about including a deprecation warning in v1alpha1?
Like:
```
deprecated: true
# This overrides the default warning returned to API clients making v1alpha1 API requests.
deprecationWarning: "example.com/v1alpha1 CronTab is deprecated; see http://example.com/v1alpha1-v1 for instructions to migrate to example.com/v1 CronTab"
    
```